homework6.py

```diff
@@ -0,0 +1,83 @@
+"""
+Homework 6.
+
+Create class representing money in different currencies.
+"""
+
+RATES = {
+    ('BYN', 'USD'): 0.47,
+    ('BYN', 'EUR'): 0.41,
+    ('BYN', 'JPY'): 51.07,
+
+    ('USD', 'BYN'): 2.15,
+    ('USD', 'EUR'): 0.88,
+    ('USD', 'JPY'): 109.75,
+
+    ('EUR', 'BYN'): 2.44,
+    ('EUR', 'USD'): 1.13,
+    ('EUR', 'JPY'): 124.48,
+
+    ('JPY', 'BYN'): 0.020,
+    ('JPY', 'USD'): 0.0091,
+    ('JPY', 'EUR'): 0.0080
+}
+
+
+class Money(object):
+    """Money class."""
+
+    def __init__(self, value, currency='USD'):
+        """Init method."""
+        self.value = value
+        self.currency = currency
+
+    def convert(self, to_cur):
+        """Currencies converting method."""
+        if self.currency != to_cur:
+            rate = RATES[(self.currency, to_cur)]
+            return self.value * rate
+        else:
+            return self.value
+
+    def __add__(self, other):
+        """+ operation method."""
+        if isinstance(other, Money):
+            if other.currency != self.currency:
+                other.value = other.convert(self.currency)
+                other.currency = self.currency
+            return self.__class__(self.value + other.value, self.currency)
```

Comment:
iti would be better to use `Money(...)` instead of  `self.__class__' as according to [PEP20](https://www.python.org/dev/peps/pep-0020/): Explicit is better than implicit. Readability counts.
However, there is a sense to use such construction for mixin class.